APIHealthMonitorWeb/src/app/home/home.component.html

```diff
@@ -0,0 +1,17 @@
+<mat-table [dataSource]="dataSource" class="mat-elevation-z8">
+
+    <ng-container [matColumnDef]="column" *ngFor="let column of tableColumns; index as i">
+        <mat-header-cell *matHeaderCellDef>{{headers[i]}}</mat-header-cell>
+        <mat-cell *matCellDef="let element">
+            <span *ngIf="column !=='createdOn'">{{element[column]}}</span>
```

Comment:
Please remove this and hardcode data as a string with data something like: '07/23/2020'